src/goto-instrument/synthesizer/loop_invariant_synthesizer.h

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************\
+
+Module: Loop Invariant Synthesizer Interface
+
+Author: Qinheping Hu
+
+\*******************************************************************/
+
+/// \file
+/// Loop Invariant Synthesizer Interface
+
+#ifndef CPROVER_GOTO_INSTRUMENT_SYNTHESIZER_LOOP_INVARIANT_SYNTHESIZER_H
+#define CPROVER_GOTO_INSTRUMENT_SYNTHESIZER_LOOP_INVARIANT_SYNTHESIZER_H
+
+#include <util/message.h>
```

Comment:
Here, you can get away with a forward declaration `class messaget;` instead, which is preferred.